Classes/Issues/Issue+IssueType.swift

```diff
@@ -11,6 +11,10 @@ import IGListKit
 
 extension IssueOrPullRequestQuery.Data.Repository.IssueOrPullRequest.AsIssue: IssueType {
 
+    var viewerSubscribed: Bool {
+        return viewerSubscription?.rawValue != "IGNORED"
```

Comment:
nit: We should use a constant for this value here: `SubscriptionState.ignored.rawValue`

Reply:
I shall fix this!